ui/src/modules/Settings/Credentials/Sections/Registry/Form.tsx

```diff
@@ -224,6 +285,16 @@ const FormRegistry = ({ onFinish }: Props) => {
     </Styled.Form>
   );
 
+  const returnRegistryIcon = () => {
```

Comment:
We usually have been using the prefix `render` or `get` to return a component

```suggestion
  const renderRegistryIcon = () => {
```